bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/spdb/cron/CronStorage.scala

```diff
@@ -0,0 +1,17 @@
+package edu.gemini.spdb.cron
+
+import java.io.File
+
+/**
+ * Storage directories for cron jobs.  Temp files don't survive updates but
+ * permanent files do.
+ */
+final case class CronStorage(tempDir: File, permDir: File) {
+
+  def newTempFile(name: String): File =
+    new File(tempDir, name)
+
+  def newPermFile(name: String): File =
```

Comment:
Should these maybe be `getXXX` rather than `newXXX` since you may end up referencing an existing file?